security_group.tf

```diff
@@ -10,7 +10,7 @@ resource "aws_security_group" "aws-demo-security-group" {
     to_port     = 0
     from_port   = 0
     protocol    = "-1"
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = ["<cidr>"]
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure no security groups allow ingress from 0.0.0.0/0 to ALL ports and protocols**
 
**Vulnerability**
It is recommended that no security group allows unrestricted ingress access. Exposed ports may allow attackers to access the application listening on that port.
 
**Remediation**
In AWS Console - 
 1. Sign in the AWS Console and go to the Amazon VPC console.
 2. In the navigation pane, select Security Groups.
 3. For each security group, perform the following actions:
      a) Select the security group.
      b) Select the Inbound Rules.
      c) Identify the rules to be removed.
      d) Remove the required rules.
 4. Click Save.

source: accurics-